src/components/DistanceRequest/DistanceRequestRenderItem.tsx

```diff
@@ -17,7 +17,7 @@ type DistanceRequestProps = {
     onSecondaryInteraction?: () => void;
 
     /** Function to get the index of the item */
-    getIndex?: () => number;
+    getIndex?: () => number | undefined;
```

Comment:
why do we need that extra `undefined` in type?

Reply:
This type is coming from the `DraggableList` (`RenderItemParams<T>`):
```typescript
export declare type RenderItemParams<T> = {
    item: T;
    getIndex: () => number | undefined;
    drag: () => void;
    isActive: boolean;
};
```